CMakeLists.txt

```diff
@@ -191,9 +191,9 @@ target_link_libraries(ni_grpc_device_server
    )
 
 if(WIN32)
-  target_link_libraries(ni_grpc_device_server Delayimp niScope niswitch nisync)
-  set_target_properties(ni_grpc_device_server PROPERTIES LINK_FLAGS "/DELAYLOAD:niScope.dll /DELAYLOAD:niswitch.dll /DELAYLOAD:nisync.dll")
-  set(ni_grpc_device_server CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS}  /DELAYLOAD:niScope.dll /DELAYLOAD:niswitch.dll /DELAYLOAD:nisync.dll")
+  target_link_libraries(ni_grpc_device_server Delayimp nidmm niScope niswitch nisync)
```

Comment:
Do these changes need to be part of this PR? I thought this should be part of the first PR which handles the autogeneration of default metadata.

Reply:
Yeah they werent supposed to be, I think I havnt pulled in the latest changes.